[BUG, ENH, MRG] Pixels (#976)
mne-tools/mne-bids: doc/whats_new.rst

- Similarly, :func:`mne_bids.get_head_mri_trans` and :func:`mne_bids.update_anat_landmarks` gained a new ``kind`` parameter to specify which of multiple landmark sets to operate on, by `Alexandre Gramfort`_ and `Richard Höchenberger`_ (:gh:`955`, :gh:`957`)

> - Add support for ieeg data in the coordinate frame 'Pixels'; although MNE does not recognize this coordinate frame and so it will be set to "unknown" in the montage, MNE can still be used to analyze this kind of data, by `Alex Rockhill`_ (:gh:`976`)

Reviewer: ```suggestion
- Add support for iEEG data in the coordinate frame ``Pixels``; although MNE-Python does not recognize this coordinate frame and so it will be set to ``unknown`` in the montage, MNE-Python can still be used to analyze this kind of data, by `Alex Rockhill`_ (:gh:`976`)
```

- :func:`mne_bids.get_head_mri_trans` now respects ``datatype`` and ``suffix`` of the provided electrophysiological :class:`mne_bids.BIDSPath`, simplifying e.g. reading of derivaties, by `Richard Höchenberger`_ (:gh:`969`)

> - Fix datasets with unknown coordinate frames being transformed to "head" by assuming an identity transform, by `Alex Rockhill`_ (:gh:`976`)

Reviewer: ```suggestion
- Do not convert unknown coordinate frames to ``head``, by `Alex Rockhill`_ (:gh:`976`)
```

> - Fix datasets with unknown coordinate frames being transformed to "head" with by assuming an identity transform, by `Alex Rockhill`_ (:gh:`976`)

Reviewer 2: ```suggestion
- Fix datasets with unknown coordinate frames being transformed to "head" by assuming an identity transform, by `Alex Rockhill`_ (:gh:`976`)
```
